### PR TITLE
Ci

### PR DESCRIPTION
**Pull request by Guillemdb**
Add basic CI for ML-mining.

### CODE REVIEW

**Comment by Guillemdb** [comment]
Everything inside the .ci folder is just a copy paste from the sourced/ci repo that Konst added to ml-core. If I need to change something in there please tell me.

**Comment by zurk** [comment]
`.ci` directory should be added to `.gitignore` 
Makefile pulls it automatically

**Comment by Guillemdb** [comment]
> `.ci` directory should be added to `.gitignore`
> Makefile pulls it automatically

https://github.com/src-d/ml-mining/pull/2/commits/408e19e25cd08edb79aad81f3e98a35764cf2902?file-filters%5B%5D=dotfile#diff-a084b794bc0759e7a6b77810e01874f2R116

 isn't it already there?

**Comment by zurk** [comment]
ok, yes. so delete all files from `.ci` directory in this PR

**Comment by Guillemdb** [comment]
Done. I also did the same thing for the `ml-core` project.

**Comment by zurk** [comment]
good! squash commits here too

File: .travis.yml [hunk 1]
```diff
@@ -11,10 +11,12 @@ before_cache:
...
   - test
-
+ - name: deploy
+  # require any tag name
+  if: tag =~ .*
 _before_install: &_before_install
-  - sudo apt-get update -y && apt install -y libsnappy-dev
-
+  - sudo apt-get update -y
+  - apt install -y libsnappy-dev
```
Comment (vmarkovtsev) [inline .travis.yml]: If this works without sudo, then we need to remove it from everywhere else

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]
Please squash all the commits

File: .travis.yml (hunk 1 above)
Comment (Guillemdb) [reply .travis.yml]: It didn't work without sudo the last time I tried. I forgot to put it in the apt intall.

**Comment by Guillemdb** [comment]
There is some mistake that prevents the CI from running

**Comment by vmarkovtsev** [comment]
@Guillemdb I see that all the tests pass except for Docker, where there is a mention of sourced.ml. `grep -R sourced.ml`; if this gives nothing, look at the base docker image.

**Review by vmarkovtsev — APPROVED** [review]

**Pull request merged**